packages/xrpl/test/models/vaultCreate.test.ts

```
@@ -0,0 +1,103 @@
+      WithdrawalPolicy: VaultWithdrawalPolicy.vaultStrategyFirstComeFirstServe,
+    }
+  })
+
+  it('verifies valid VaultCreate', function () {
+    assert.doesNotThrow(() => validateVaultCreate(tx))
+    assert.doesNotThrow(() => validate(tx))
+  })
+
+  it('throws w/ missing Asset', function () {
+    // @ts-expect-error for test
+    delete tx.Asset
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)